src/modules/person/person.module.ts

```diff
@@ -21,7 +22,8 @@ import { EventModule } from '../../core/eventbus/event.module.js';
         OrganisationRepository,
         RolleFactory,
         ServiceProviderRepo,
+        OxUserBlacklistRepo,
     ],
-    exports: [PersonService, PersonFactory, PersonRepository],
+    exports: [PersonService, PersonFactory, PersonRepository, OxUserBlacklistRepo],
```

Comment:
Warum brauchst du OxUserBlacklistRepo in  exports?

Reply:
Villeicht liege ich falsch aber das reicht doch wenn es nur in den Providers steht, man importiert sowieso das gesammte Modul PersonModule in anderen Modulen, die UsernameGeneratorService & OxUserBlacklistRepo Abhängigkeiten haben. Ich verstehe das so dass wenn ein Provider exportiert wird, braucht man das explizit z.B: in den Controllers wenn z.B.: PersonApiModule PersonModule importiert.

Reply:
Ich denke jetzt ist es gerade gezogen